dags/atd_moped_components_to_agol.py

```diff
@@ -43,30 +45,94 @@
 }
 
 
+@task(
+    task_id="get_args",
+)
+def get_args(params, **context):
```

Comment:
This new helper is very specific to this DAG. We used to recycle `get_date_filter_arg` which was written for our `atd-knack-services` DAGs and is a better fit there since this one never used the `should_replace_monthly` option  it has.

Reply:
late to the party, but it is very awesome to see this in action and know that we have a pattern for it now! 🚀